landing/src/app/page.tsx

```diff
@@ -0,0 +1,225 @@
+import {DocsFooter, DocsNavBar, DocsTripleCard, DocsHeaderWrapper, DocsHeaderLeft, DocsHeaderRight, DocsHeaderSubtitle, Button, DocsPageLayout, CodeCopy, Card, CardHeader, CardTitle, CardDescription } from '@cqcl/quantinuum-ui'
+import { FaGithub, FaDiscord } from 'react-icons/fa'
+import { LifeBuoyIcon, BookIcon } from "lucide-react";
+
+import Image from 'next/image'
+import Link from 'next/link'
+import { LambeqLogo } from './LambeqLogo';
+
+const navConfig = {
+  navTextLinks: [
+    {
+      title: 'Getting Started',
+      href: '/intro.html',
```

Comment:
I guess all these need to be `docs/intro.html`, `docs/pipeline.html' etc? Locally none of these links work.

Reply:
works for me locally